services/EmailService.ts

```diff
@@ -88,6 +90,14 @@ export default class EmailService {
           pickupEvent: order.pickupEvent,
           link: `${Config.client}/store/orders`,
         }),
+        attachments: [
+          {
+            content: EmailService.createCalendarFile(calendarInfo),
+            filename: 'invite.ics',
+            type: 'text/calendar',
```

Comment:
Seeking to learn: Where does this type come from? Is this just how ics files are meant to be defined (is this a Sendgrid/EJS/ics thing?)

Reply:
This is the mime type, it is used to identify the file format. For example we use "application/json" to send out the responses from the backend in json format